index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="style.css" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+       
+      
+       <section class="headline">
+       
+        <header class="toolbar">
+            <a href="#" ><p>My logo </p></a>
+            <div class ="vector">
+                <a href="#" >   <img src = "/Icons/Union.png"> 
```

Comment:
- [ ] Please update this path to a relative path as your button is not visible now to all browsers. 
![image](https://user-images.githubusercontent.com/91486791/219351094-3dd0a00b-0e3e-41ea-85f2-a0b575c5525a.png)